packages/tools/devtools/devtools-core/src/data-visualization/SharedTreeHelpers.ts

```diff
@@ -0,0 +1,204 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/core-utils";
+import type {
+	FieldMapObject,
+	JsonableTree,
+	SharedTreeContentSnapshot,
+	TreeNodeStoredSchema,
+} from "@fluidframework/tree/internal";
+import {
+	LeafNodeStoredSchema,
+	MapNodeStoredSchema,
+	ObjectNodeStoredSchema,
+} from "@fluidframework/tree/internal";
+import type { SharedTreeLeafNode, VisualSharedTreeNode } from "./VisualSharedTreeTypes.js";
+import { type VisualChildNode, VisualNodeKind, type VisualValueNode } from "./VisualTree.js";
+
+/**
+ * TODO
+ */
+export function visualRepresentationMapper(tree: VisualSharedTreeNode): VisualChildNode {
```

Comment:
I would recommend naming functions as verbs. And here, we should call out what it does. E.g. `toVisualTree` or something like that.